### PR TITLE
[tests-only] [full-ci] Add test coverage for copying file using file id from `project space`

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
## Description
Added test coverage for copying a file using fileId from project space:
| |from (space) | from (role) | to (space) | to (role) 
--|--|--|--|--|
| 1. | project | different permission | project | different permission 
| 2. | project | different permission | share | different permission
| 3. | project | different permission	| personal

## Related Issue
- Part of https://github.com/owncloud/ocis/issues/9878

## How Has This Been Tested?
- locally
- CI :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature [hunk 1]
```diff
@@ -280,3 +280,192 @@ Feature: copying file using file id
...
+      | Manager      | /remote.php/dav/spaces/<<FILEID>> |
+      | Space Editor | /remote.php/dav/spaces/<<FILEID>> |
+      | Space Viewer | /remote.php/dav/spaces/<<FILEID>> |
+      | Manager      | /dav/spaces/<<FILEID>>            |
+      | Space Editor | /dav/spaces/<<FILEID>>            |
+      | Space Viewer | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: sharee copies a file between two project spaces
+    Given user "Brian" has been created with default attributes and without skeleton files
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:313]: what about similar scenario? maybe delete it or update it?
https://github.com/owncloud/ocis/blob/14e0a6a58828a47cf3d248b8996144ab0368dacc/tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature#L269

Comment (Salipa-Gurung) [reply tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:313]: Is it okay if I remove scenario: `Scenario Outline: copy a file between two project spaces` as copy (between two project spaces) feature will be tested in scenario: `Scenario Outline: sharee copies a file between two project spaces`? 
Though, scenario: `Scenario Outline: sharee copies a file between two project spaces` is for received project space it shouldn't matter I suppose.

File: tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature [hunk 2]
```diff
@@ -280,3 +280,192 @@ Feature: copying file using file id
       | dav-path                          |
       | /remote.php/dav/spaces/<<FILEID>> |
       | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: sharee copies a file from project to personal space
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:285]: in project context, let's not use sharee
```suggestion
  Scenario Outline: copy a file from project to personal space
```

File: tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature (hunk 1 above)
Comment (saw-jan) [reply tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:313]: yes, you should delete the scenario and add this one.

File: tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature [hunk 3]
```diff
@@ -266,27 +249,7 @@ Feature: copying file using file id
...
-    And for user "Alice" folder "/" of the space "second-project-space" should contain these files:
-      | secondProjectSpacetextfile.txt |
-    Examples:
-      | dav-path                          |
-      | /remote.php/dav/spaces/<<FILEID>> |
-      | /dav/spaces/<<FILEID>>            |
-
-
-  Scenario Outline: sharee tries to copy a file from shares space with secure viewer to personal space
+  Scenario Outline: try to copy a file from shares space with secure viewer to personal space
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:252]: old one suites better here
```suggestion
  Scenario Outline: sharee tries to copy a file from shares space with secure viewer to personal space
```

File: tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature [hunk 4]
```diff
@@ -310,7 +273,7 @@ Feature: copying file using file id
       | /dav/spaces/<<FILEID>>            |
 
 
-  Scenario Outline: sharee copies a file from shares to project space
+  Scenario Outline: copy a file from shares to project space
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:276]:
```suggestion
  Scenario Outline: sharee copies a file from shares to project space
```

File: tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature [hunk 5]
```diff
@@ -353,7 +316,7 @@ Feature: copying file using file id
       | Uploader        | Space Editor  | /dav/spaces/<<FILEID>>            |
 
 
-  Scenario Outline: sharee tries to copy a file from shares to project space
+  Scenario Outline: try to copy a file from shares to project space
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:319]:
```suggestion
  Scenario Outline: sharee tries to copy a file from shares to project space
```

File: tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature [hunk 6]
```diff
@@ -396,7 +359,7 @@ Feature: copying file using file id
       | Uploader        | Space Viewer  | /dav/spaces/<<FILEID>>            |
 
 
-  Scenario Outline: sharee copies a file between shares spaces
+  Scenario Outline: copy a file between shares spaces
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:362]:
```suggestion
  Scenario Outline: sharee copies a file between shares spaces
```

File: tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature [hunk 7]
```diff
@@ -442,7 +405,7 @@ Feature: copying file using file id
       | Uploader        | Uploader      | /dav/spaces/<<FILEID>>            |
 
 
-  Scenario Outline: sharee tries to copy a file between shares space
+  Scenario Outline: try to copy a file between shares space
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature:408]:
```suggestion
  Scenario Outline: sharee tries to copy a file between shares space
```

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Pull request merged**